Create ui-c8y-1021-22-86-clear-cookies-when-401-is-received-on-current-tenant-on-login.md (#3387)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-22-86-clear-cookies-when-401-is-received-on-current-tenant-on-login.md

---
date: ""

> title: Clear cookies when 401 is received on currentTenant on login (#9189)

Reviewer: ```suggestion
title: SSO Login Issues after upgrade - automatic cookie cleanup implemented
```

> title: SSO Login Issues after upgrade - automatic cookie cleanup implemented

Reviewer 2: ```suggestion
title: Fixed SSO login issues after upgrade
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63747
version: 1021.22.86
---

> For tenants using single sign-on as a default login method, after an upgrade from y2024 release (or earlier) to y2025 release (or newer) user logins might fail.

Reviewer 2: ```suggestion
For tenants using single sign-on as a default login method, after an upgrade from the y2024 release (or earlier) to the y2025 release (or newer) user logins might fail.
```

> For tenants using single sign-on as a default login method, after an upgrade from the y2024 release (or earlier) to the y2025 release (or newer) user logins might fail.

Reviewer: ```suggestion
For tenants using single sign-on as a default login method, after an upgrade from the y2024 release (or earlier) to the y2025 release user logins might fail.
```

> For tenants using Single Sign On-redirect as a default login method, after an upgrade from y2024 release (or earlier) to y2025 release (or newer) user logins might fail.

Reviewer 2: ```suggestion
For tenants using single sign-on as a default login method, after an upgrade from y2024 release (or earlier) to y2025 release (or newer) user logins might fail.
```

> The reason is that older versions were using authorization token for subdomains (e.g. ".test.eu-latest.cumulocity.com") and newer one are using domain specific token ("test.eu-latest.cumulocity.com", without dot at the beginning) and presence of them both was causing conflict. Currently, if old cookie is detected, cookies are cleared with logout and users can log in again successfuly.

Reviewer 2: ```suggestion
The reason is that older versions used authorization tokens for subdomains (for example, *.test.eu-latest.cumulocity.com*) while newer versions use domain-specific tokens (*test.eu-latest.cumulocity.com*, without dot at the beginning). Having both in place was causing a conflict. Now, if an old cookie is detected, cookies are cleared with logout and users can log in again without issues.
```
